### ISSUE TITLE
Cannot deploy multiple trino clusters in the same k8s namespace

### ISSUE THREAD

**Issue by sudohainguyen** [trinodb/charts#159]
### Describe the bug
Not able to deploy multiple clusters using helm chart due to configmap conflicts (caused by this [line](https://github.com/trinodb/charts/blob/50e5089cb26a670f52d396e15ff3f3ffa4bceb7b/charts/trino/templates/configmap-coordinator.yaml#L152) for example)

chart version: 0.20.0

### Suggestion
Templating configmap objects name using predefined template in `_helpers.tpl`
e.g. 
instead of `trino-resource-groups-volume-coordinator`
use `trino-resource-groups-volume-{{ template trino-resource-groups-volume . }}`

**Comment by nineinchnick** [comment]
Why do you need to install multiple clusters into a single namespace? Can you use separate namespaces?

**Comment by sudohainguyen** [comment]
I want to setup multiple clusters to serve different workloads (interactive vs BI reports), think I should reuse existing datalake access permission setup in the namespace instead of duplicating to another one

**Comment by nineinchnick** [comment]
I'd recommend using separate namespaces. Can you share some more details on how the access permissions for the Kubernetes namespace are configured? It's independent of any Trino configuration, right?

**Comment by mosabua** [comment]
I think it actually would make more sense to have separate namespaces for the separate clusters.. that way you can control resources per cluster and usecase better .. since they are isolated in the namespaces.

**Comment by silent-lad** [comment]
I also landed on this problem. It works good with fullnameOverride for some resources but configmaps like `trino-resource-groups-volume-coordinator` have fixed names.
Our usecase is to deploy multiple trino clusters sitting behind a trino-gateway. The clusters are identical in work load and configuration.

**Comment by sudohainguyen** [comment]
yeah I think the use case is not quite unusual though, think we should let chart users decide which strategy to employ

**Comment by mgorbatenko** [comment]
For what it is worth, I also have a use case where I would like to deploy multiple clusters in the same namespace. Similar to @silent-lad, we are deploying multiple clusters behind a gateway. The clusters can be different in configuration, but we have other ways of identifying resources assigned to a particular cluster. It's much easier for our use case to have everything in one namespace from a few different perspectives.
